components/fxa-client/ios/FxAClient/FxAccountDevice.swift

```diff
@@ -115,11 +115,11 @@ public enum IncomingDeviceCommand {
     internal static func fromMsg(msg: MsgTypes_IncomingDeviceCommand) -> IncomingDeviceCommand {
         switch msg.type {
         case .tabReceived: do {
-            let data = msg.tabReceivedData
-            let device = data.hasFrom ? Device(msg: data.from) : nil
-            let entries = data.entries.map { TabData(title: $0.title, url: $0.url) }
-            return .tabReceived(device, entries)
-        }
+                let data = msg.tabReceivedData
+                let device = data.hasFrom ? Device(msg: data.from) : nil
+                let entries = data.entries.map { TabData(title: $0.title, url: $0.url) }
+                return .tabReceived(device, entries)
+            }
```

Comment:
Suddenly getting swiftlint failures for fxa-component, maybe a new version of swiftlint come out or something? Anyway, fixed them inline here to reduce review load.

Reply:
huh - yeah, I saw them on @lougeniaC64's branch and was confused!